### ISSUE TITLE
So you at Realme built all linux kernel by yourself??

### ISSUE THREAD

**Issue by Thatisjigen** [realme-kernel-opensource/realme7_narzo20pro_AndroidQ-kernel-source#2]
Please leave us know what caf tag are you basing your sources, you know, like any other OEM does?

**Comment by SALMANKHANPM** [comment]
> Please leave us know what caf tag are you basing your sources, you know, like any other OEM does?
> I mean, you still can make one commit for all your modifications, but atleast preserve a decent history.

yes ! they should !

**Comment by LinkBoi00** [comment]
Yeah realmeme Devs are pros dude. They even managed to run a CAF source on an MTK device lmfao, according to them at least.

Meanwhile, you can check this out: 
https://forum.xda-developers.com/t/guide-working-with-android-kernel-from-scratch.3909887/
